src/pages/Proposal/index.spec.tsx

```diff
@@ -133,12 +134,79 @@ describe('pages/Proposal', () => {
     const { getByTestId } = renderComponent(<Proposal />, {
       authContextValue: {
         accountAddress: fakeAddress,
+        chainId: ChainId.BSC_TESTNET,
       },
     });
 
     await checkAllButtons(getByTestId, (element: HTMLElement) => expect(element).toBeEnabled());
   });
 
+  it('vote buttons are not enabled on ETHEREUM chain', async () => {
```

Comment:
To follow up with my previous comment, I'd change the scope of the tests so that we don't check it on every chain, but instead check it base on what `useIsFeatureEnabled` returns. This way we don't check the implementation of `useIsFeatureEnabled` itself .